routes/mapcss/post.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = {
+    method: 'POST',
+    path: '/mapcss',
+    config: {
+        handler: require('../../handlers/mapcss').post,
+        cors: { origin: ['*'], additionalHeaders: ['cache-control', 'x-request-with'] }
+    }
```

Comment:
given the note in the handler file, we'll want to add some sort of response validation, with a new property

`config.response.schema`

I think we can just make use of some of the schema goodness found in the mapcss-parse dependency to make sure that every response is an array of object where each object has at least the minimum 3 properties of a mapcss json, the geometry, selector, and the throw type (error or warning)